src/sentry/static/sentry/app/views/monitors/monitorHeaderActions.jsx

```diff
@@ -71,14 +72,15 @@ class MonitorHeaderActions extends React.Component {
         <div className="btn-group">
           <Button
             size="small"
-            icon="icon-edit"
             to={`/organizations/${orgId}/monitors/${monitor.id}/edit/`}
           >
+            <IconEdit size="xs" />
+            &nbsp;
             {t('Edit')}
           </Button>
         </div>
         <div className="btn-group" style={{marginLeft: 10}}>
-          <Button size="small" icon="icon-edit" onClick={this.toggleStatus}>
+          <Button size="small" onClick={this.toggleStatus}>
```

Comment:
This button lost its icon.

Reply:
I put it in the button - I realized we were already doing that in a few places

Reply:
It actually didn't make sense to put an edit icon in this since it's the pause button